src/main/java/bio/terra/janitor/service/cleanup/CleanupParams.java

```diff
@@ -0,0 +1,25 @@
+package bio.terra.janitor.service.cleanup;
+
+import bio.terra.generated.model.CloudResourceUid;
+import bio.terra.janitor.db.TrackedResourceId;
+import bio.terra.stairway.FlightMap;
+
+/** /** Helper class to manage input parameters in a {@link FlightMap}. */
+public final class CleanupParams {
+  public static final String TRACKED_RESOURCE_ID = "trackedResourceId";
+  public static final String CLOUD_RESOURCE_UID = "cloudResourceUid";
```

Comment:
I think these should be private and we should have `set` methods.

Reply:
I'd say let's use it for `get` (in the steps) and `set` (in flight submission construction) or not at all. Either we're encapsulating these flightmap string keys, or we're not.